presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveFileSystem.java

```diff
@@ -384,20 +386,21 @@ public void testRename()
     public void testTableCreation()
             throws Exception
     {
-        for (HiveStorageFormat storageFormat : HiveStorageFormat.values()) {
-            if (storageFormat == HiveStorageFormat.CSV) {
-                // CSV supports only unbounded VARCHAR type
-                continue;
-            }
-            if (storageFormat == HiveStorageFormat.ALPHA) {
-                // Alpha read/write is not supported yet
-                continue;
-            }
+        for (HiveStorageFormat storageFormat : getSupportedHiveStorageFormats()) {
             createTable(METASTORE_CONTEXT, temporaryCreateTable, storageFormat);
             dropTable(temporaryCreateTable);
         }
     }
 
+    protected List<HiveStorageFormat> getSupportedHiveStorageFormats()
+    {
+        // CSV supports only unbounded VARCHAR type, and Alpha does not support DML yet
+        return Arrays.stream(HiveStorageFormat.values())
+                .filter(format -> format != HiveStorageFormat.CSV)
+                .filter(format -> format != HiveStorageFormat.ALPHA)
```

Comment:
nit : We can merge the filters in one line
.filter(format -> format != HiveStorageFormat.CSV && format != HiveStorageFormat.ALPHA)

Reply:
Done